docs/remote/devcontainer-cli.md

```diff
@@ -1,220 +1,140 @@
 ---
-Order: 13
+Order: 14
 Area: remote
 TOCTitle: devcontainer CLI
 PageTitle: Installing and working with the devcontainer CLI
 ContentId: 8946213d-716e-41ca-955f-944a41c70353
-MetaDescription: Documentation on using the VS Code development container (devcontainer) command line interface with the Visual Studio Code Remote - Containers extension
+MetaDescription: Documentation on using the development container (devcontainer) command-line interface
 DateApproved: 7/7/2022
 ---
 # Development container CLI
 
-When we refer to a command line interface (CLI) for development containers, there are two varieties:
+This topic covers the development container command-line interface (devcontainer CLI), which allows you to build and manage development containers, and is a companion to the [Development Containers Specification](https://containers.dev).
 
-* "dev container CLI" - A reference implementation for the open dev container specification. The [current proposal](https://github.com/microsoft/dev-container-spec/issues/9) is to make a CLI available that can take a `devcontainer.json` and create and configure a development container from it. It could be available in any IDE or editor.
-* "Visual Studio Code `devcontainer` CLI" - A CLI that can be installed and used via the [Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) or through an external terminal.
+## Development containers
 
-The former is in-progress. The latter, VS Code `devcontainer` CLI, is available today and the focus of this document.
+A consistent, predictable environment is key to a productive and enjoyable software development experience.
 
-## Visual Studio Code devcontainer CLI
+Containers (such as [Docker](https://www.docker.com)) have historically been used to standardize apps when they're deployed, but there's a great opportunity to support additional scenarios, including continuous integration (CI), test automation, and full-featured coding environments. A **development container** provides this working environment and ensures your project has the tools and software it needs, whether it's complex and distributed or just has a few requirements.
```

Comment:
Docker isn't really a container itself but rather an engine/provider of containers. Maybe we amend to something like the following, or just remove mention of it?

 ```suggestion
Containers (such as those provided by [Docker](https://www.docker.com)) have historically been used to standardize apps when they're deployed, but there's a great opportunity to support additional scenarios, including continuous integration (CI), test automation, and full-featured coding environments. A **development container** provides this working environment and ensures your project has the tools and software it needs, whether it's complex and distributed or just has a few requirements.
```

Reply:
I wanted to have a concrete example in case someone has no idea what Containers are.
Docker provides but also lets you create containers, so how about "Containers (for example [Docker](link) containers) have"